{
    "original_source": {
        "database(url)": "https://www.cve.org",
        "software_system(CPE)": [
            "cpe:2.3:a:yetiforce:yetiforce_customer_relationship_management:*:*:*:*:*:*:*:*"
        ],
        "fix_commit": "https://github.com/yetiforcecompany/yetiforcecrm/commit/e55886781509fe39951fc7528347696474a17884",
        "version_end_excluding": {
            "yetiforce:yetiforce_customer_relationship_management": "6.4.0"
        },
        "path": {
            "file_path": ""
        }
    },
    "programming_language": "JSON",
    "licences": {
        "database": "",
        "affected_repository": "Other"
    },
    "related_vulnerability_entry": {
        "CVE": [
            "CVE-2022-3005"
        ],
        "CWE": [
            "CWE-79"
        ]
    }
}